compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1737,4 +1737,12 @@ object messages {
       hl"you have to provide either ${"class"}, ${"trait"}, ${"object"}, or ${"enum"} definitions after qualifiers"
   }
 
+  case class ReturnOutsideMethodDefinition()(implicit ctx: Context)
+    extends Message(ReturnOutsideMethodDefinitionID) {
+    val kind = "Syntax"
+    val msg = s"return outside method definition"
```

Comment:
I would turn the phrasing around and say that `retrun` must be inside a method definition.

Reply:
I kept the wording from the original, which is the same in scalac.

Reply:
Ok

Reply:
Though we can improve